contracts/apps/disputable/DisputableApp.sol

```diff
@@ -0,0 +1,186 @@
+    * @param _disputableActionId Identification number of the disputable action to be rejected
+    */
+    function onDisputableActionRejected(uint256 _disputableActionId) external onlyAgreement {
+        _onDisputableActionRejected(_disputableActionId);
+    }
+
+    /**
+    * @notice Void disputable action #`_disputableActionId`
+    * @param _disputableActionId Identification number of the disputable action to be voided
+    */
+    function onDisputableActionVoided(uint256 _disputableActionId) external onlyAgreement {
+        _onDisputableActionVoided(_disputableActionId);
+    }
+
+    /**
+    * @notice Tells whether the Agreement app is a forwarder or not
+    * @dev IForwarder interface conformance
+    * @return Always true
+    */
+    function isForwarder() external pure returns (bool) {
+        return true;
+    }
+
+    /**
+    * @dev Tell the agreement linked to the disputable instance
+    * @return Agreement linked to the disputable instance
+    */
+    function getAgreement() external view returns (IAgreement) {
+        return _getAgreement();
+    }
+
+    /**
+    * @dev Create a new action in the agreement without lifetime set
+    * @param _disputableActionId Identification number of the disputable action in the context of the disputable
+    * @param _submitter Address of the user that has submitted the action
+    * @param _context Link to a human-readable text giving context for the given action
+    * @return Unique identification number for the created action in the context of the agreement
+    */
+    function _newAgreementAction(uint256 _disputableActionId, address _submitter, bytes _context) internal returns (uint256) {
+        return _newAgreementAction(_disputableActionId, 0, _submitter, _context);
+    }
+
+    /**
+    * @dev Create a new action in the agreement
+    * @param _disputableActionId Identification number of the disputable action in the context of the disputable
+    * @param _lifetime Lifetime duration in seconds of the disputable action, it can be set to zero to specify infinite
+    * @param _submitter Address of the user that has submitted the action
+    * @param _context Link to a human-readable text giving context for the given action
+    * @return Unique identification number for the created action in the context of the agreement
+    */
+    function _newAgreementAction(uint256 _disputableActionId, uint64 _lifetime, address _submitter, bytes _context) internal returns (uint256) {
+        IAgreement agreement = _ensureAgreement();
+        return agreement.newAction(_disputableActionId, _lifetime, _submitter, _context);
+    }
+
+    /**
+    * @dev Close action in the agreement
+    * @param _actionId Identification number of the disputable action in the context of the agreement
+    */
+    function _closeAgreementAction(uint256 _actionId) internal {
+        IAgreement agreement = _ensureAgreement();
+        agreement.closeAction(_actionId);
+    }
+
+    /**
```

Comment:
Maybe we should lift this must-implement functions up to the top of the contract, or separate them into another interface, e.g. `DisputableAppImplementation`?

I'd like to make it obvious to app developers that they need to implement these when working with `DisputableApp`, and why they're implementing these instead of the `IDisputable` interface's functions.

Reply:
Not sure if I'd add another interface for this. Currently, it is "obvious" since simply inheriting from this contract won't compile, developers must implement these methods. 

I lifted them to the top to make it clearer